tdrs-backend/tdpservice/settings/common.py

```diff
@@ -435,23 +435,23 @@ class Common(Configuration):
     CELERY_BEAT_SCHEDULE = {
         'name': {
             'task': 'tdpservice.scheduling.tasks.postgres_backup',
-            'schedule': crontab(minute='*', hour='4'), # Runs at midnight EST
+            'schedule': crontab(minute='0', hour='5'), # Runs at midnight EST
             'args': "-b",
             'options': {
                 'expires': 15.0,
             },
         },
         'name': {
             'task': 'tdpservice.scheduling.tasks.check_for_accounts_needing_deactivation_warning',
-            'schedule': crontab(day_of_week='*', hour='13', minute='*'), # Every day at 1pm UTC (9am EST)
+            'schedule': crontab(day_of_week='*', hour='13', minute='0'), # Every day at 1pm UTC (8am EST)
```

Comment:
@elipe17 

- i think hour 13 UTC is 9am EST? 
- i changed the crontab that in the screenshot that has the arrow pointing to it and monitored the the logs in cloud.gov. which looks good!
![2505p1](https://github.com/raft-tech/TANF-app/assets/63075587/c5bf5a44-a3fd-41de-96fe-1a653f3157ea)
![2505p2](https://github.com/raft-tech/TANF-app/assets/63075587/8fd1d0e0-0ce4-4a4c-ac61-39090399a4a3)

- for the highlighted cron schedule, can you verify that we should go ahead and delete that one since it looks like it isnt deleted automatically?

Reply:
I updated the labels. I had EDT instead of EST. But that is correct, the highlighted one can be deleted if it is not replaced by the new crontab (not sure why it isn't deleted/updated automatically).